libs/features/creature/src/creature-text/creature-text.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { MultiRowEditorComponent } from '@keira/shared/base-abstract-classes';
+import { CreatureText } from '@keira/shared/acore-world-model';
+import { CreatureHandlerService } from '../creature-handler.service';
+import { CreatureTextService } from './creature-text.service';
+import { NgxDatatableModule } from '@swimlane/ngx-datatable';
+import { EditorButtonsComponent } from '@keira/shared/base-editor-components';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { QueryOutputComponent } from '@keira/shared/base-editor-components';
+import { TranslateModule } from '@ngx-translate/core';
+import { TopBarComponent } from '@keira/shared/base-editor-components';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'keira-creature-text',
+  templateUrl: './creature-text.component.html',
+  styleUrls: ['./creature-text.component.scss'],
+  standalone: true,
+  imports: [
+    TopBarComponent,
+    TranslateModule,
+    QueryOutputComponent,
+    FormsModule,
+    ReactiveFormsModule,
+    EditorButtonsComponent,
+    NgxDatatableModule,
+  ],
+})
+export class CreatureTextComponent extends MultiRowEditorComponent<CreatureText> {
+  readonly editorService = inject(CreatureTextService);
+  readonly handlerService = inject(CreatureHandlerService);
```

Comment:
you could set these as protected

Reply:
editorService in public in parent class MultiRowEditorComponent. So that needs to stay public, but I will change handlerService.